src/components/select/select.tsx

```diff
@@ -205,7 +209,7 @@ export class Select {
       (<fw-select-option
         value={option.value}
         selected={option.selected}
-        disabled={option.disabled}
+        disabled={option.disabled || (this.value?.length >= this.max)}
```

Comment:
Should we disable `fw-select` or `fw-select-option` @asifahmedfw ?

Reply:
If you disable the entire component, you can't remove the items